atlp-rwanda/ecommerce-app-champions-bn
# ft: roles permissions

#### What does this PR do?
-  implement roles and permissions
#### Description of Task to be completed
- implement create permissions
- implement create roles
- implement assign role and permissions to the user being created
#### How should this be manually tested?
- clone the repo using git clone repo's URL
- change branch git checkout ft-roles-permissions
- create a .env file and put the required environment variables from .env.example file
- run in the terminal `npm run migrate:up`,
- make a post request `http://localhost:5000/api/permission/create` to create permission
- make a post request `http://localhost:5000/api/role/create` to create role
- make a post request `http://localhost:5000/api/vendor/signup` to create a user  , the roles and permissions are automatically assigned
- to check the roles and permissions assigned to the user `http://localhost:5000/api/vendor/{id}`
#### What are the relevant pivotal tracker stories?
-Create Roles and Set Permissions `card #12`
